qe-tests/cypress/integration/tests/vmware/config_separate_mapping.ts

```diff
@@ -14,6 +15,8 @@ const v2v_vmware_username = Cypress.env('v2v_vmware_username');
 const v2v_vmware_password = Cypress.env('v2v_vmware_password');
 const v2v_vmware_hostname = Cypress.env('v2v_vmware_hostname');
 const v2v_vmware_cert = Cypress.env('v2v_vmware_cert');
+const preAnsiblePlaybook = Cypress.env('ansiblePlaybook');
+const postAnsiblePlaybook = Cypress.env('ansiblePlaybook_1');
```

Comment:
env variable name `ansiblePlaybook_1` is not acceptable, please update it to something like `postAnsiblePlaybook`